substra/sdk/backends/local/models.py

```diff
@@ -109,17 +127,22 @@ def __str__(self):
 
 
 class DataSample(_Model):
```

Comment:
Are the changes logic ?
- we could instead remove "owner"
- do the data sample keep the list of the data manager keys ? The remote backend does not return it

Reply:
Yes, in this case we could remove it.